targets/TARGET_STM/TARGET_STM32F0/objects.h

```diff
@@ -77,39 +77,6 @@ struct serial_s {
 #endif
 };
 
-struct i2c_s {
```

Comment:
It's a bit silly that this structure is copied verbatim into like 10 different objects.h headers, despite all of them being identical and being used by a single version of i2c_api.c.  Since I had to modify the structure, I consolidated these into a single version in `stm_i2c_api.h` while I was at it.